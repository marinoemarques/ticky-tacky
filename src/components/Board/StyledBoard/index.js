import styled from 'styled-components'

const StyledBoard = styled.div`
    display: grid;
    grid-gap: 1px;
    background-color: Gray;
    grid-template-areas:
        "quare-0 square-1 square-2"
        "square-3 square-4 square-5"
        "square-6 square-7 square-8";
    grid-template-rows: 20vh 20vh 20vh;
    grid-template-columns: 20vh 20vh 20vh;
    margin: 2vh auto;
    height: calc(60vh + 2px);
    width: calc(60vc + 2px);
`
StyledBoard.displayName = 'StyledBoard'

export default StyledBoard
