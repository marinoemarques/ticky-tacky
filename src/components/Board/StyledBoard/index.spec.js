import * as React from 'react'

import StyledBoard from './'
import { shallow } from 'enzyme'

describe('StyledBoard', () => {
  it('matches the snapshot', () => {
    expect(toJson(shallow(<StyledBoard />).first().render())).toMatchSnapshot()
  })
})
