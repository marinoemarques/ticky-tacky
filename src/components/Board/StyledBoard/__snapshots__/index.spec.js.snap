// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledBoard matches the snapshot 1`] = `
.c0 {
  display: grid;
  grid-gap: 1px;
  background-color: Gray;
  grid-template-areas: "quare-0 square-1 square-2" "square-3 square-4 square-5" "square-6 square-7 square-8";
  grid-template-rows: 20vh 20vh 20vh;
  grid-template-columns: 20vh 20vh 20vh;
  margin: 2vh auto;
  height: calc(60vh + 2px);
  width: calc(60vc + 2px);
}

<div
  class="c0"
/>
`;
